src/mount/sauna_client.cc

```diff
@@ -858,10 +877,12 @@ AttrReply getattr(Context &ctx, Inode ino) {
 		}
 		stats_inc(OP_DIRCACHE_GETATTR);
 		status = SAUNAFS_STATUS_OK;
+		fromcache = 1;
```

Comment:
Consider using true/false in the assignment for the bool variable.

Reply:
Done.